src/PageLayout/PageLayout.tsx

```diff
@@ -736,6 +736,7 @@ const Pane = React.forwardRef<HTMLDivElement, React.PropsWithChildren<PageLayout
               '--pane-max-width-diff': '959px',
             },
           })}
+          {...(id && {id: paneId})}
```

Comment:
How do you feel about simplifying to this?

```suggestion
          id={id}
```

If `id` is `undefined`, it won't pass `id` to the underlying component.

If `id` is defined, `paneId` and `id` will be the same, so we can use either.